cdk/lib/frontend.ts

```diff
@@ -0,0 +1,22 @@
+import { join } from "path";
+import { CfnInclude } from "@aws-cdk/cloudformation-include";
```

Comment:
This is the file that we'll actually be editing to replace our old cloudformation with `cdk` constructs. For now, we're just using `CfnInclude` to directly load our existing cloudformation template